test/passing/break_cases_all.ml

```diff
@@ -4,80 +4,90 @@ let f x = function
   | C
   | P (this, test, [is; wide; enough; _to; break], [the; line])
   | A
-  | K ->
-      1
+   |K -> 1
   | D ->
       let a = "this" in
       let b = "breaks" in
       ()
 
 let f =
   let g = function
-    | H
-      when x y <> k ->
-        2
+    | H when x y <> k -> 2
     | T
-    | P
-    | U ->
-        3
+     |P
+     |U -> 3
   in
   fun x g t h y u ->
     match x with
-    | E ->
-        4
+    | E -> 4
     | Z
-    | P
-    | M -> (
+     |P
+     |M -> (
       match y with
-      | O ->
-          5
-      | P
-        when h x -> (
+      | O -> 5
+      | P when h x -> (
           function
-          | A ->
-              6 ) )
+          | A -> 6 ) )
 
 ;;
 match x with
 | true -> (
   match y with
-  | true ->
-      "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
-  | false ->
-      "bbbbbbbbbbbbbbbbbbbbbbbbbbbbbb" )
-| false ->
-    "cccccccccccccccccccccccccccccc"
+  | true -> "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+  | false -> "bbbbbbbbbbbbbbbbbbbbbbbbbbbbbb" )
+| false -> "cccccccccccccccccccccccccccccc"
 
 ;;
 match x with
 | "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa", yyyyyyyyyy
-  when fffffffffffffff bbbbbbbbbb yyyyyyyyyy ->
-    ()
-| _ ->
-    ()
+  when fffffffffffffff bbbbbbbbbb yyyyyyyyyy -> ()
+| _ -> ()
 
 let is_sequence exp =
   match exp.pexp_desc with
   | Pexp_sequence _
   | Pexp_extension
```

Comment:
Is `indicate-nested-or-patterns` false for some reason here? If not, this case looks like it is not respecting it.

Reply:
fixed, thanks